### ISSUE TITLE
update releases and records

### ISSUE THREAD

**Issue by kMutagene** [nfdi4plants/isa-ro-crate-profile#31]

**Comment by HLWeil** [comment]
Done with https://github.com/nfdi4plants/isa-ro-crate-profile/releases/tag/1.0.0-draft.1?

**Comment by kMutagene** [comment]
Yes i think so. This was meant as a reminder to unify formatting of the releases and zenodo records. I think the releases should be less structured like software release notes that point to issues/PRs, and be more complete as a self-contained text. I think the current one is good, but i was also thinking about re-writing the old ones, but no time for that atm so lets close
